### PR TITLE
refactored 

### PR DESCRIPTION
**Pull request by saketkum11**
- add toast
- login / logout on side nav handle

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
+import { Toaster } from "react-hot-toast";
 import "./App.css";
+
 import { MyRoutes } from "./Pages";
 function App() {
   return (
     <div className="App">
       <MyRoutes></MyRoutes>
```
Comment (sharathnair9999) [inline src/App.js]: if nothing in between, put it as a self closing tag

Comment (MiheerTamkhane) [inline src/App.js]: You can use self closing tag here..

File: src/Component/AddPlaylist/AddPlaylist.jsx [hunk 2]
```diff
@@ -1,12 +1,30 @@
 import { useState } from "react";
 import { usePlay } from "../../Context/Playlets-context/Playlets-context";
 import { useVideo } from "../../Context/Video-Context/video-context";
+import { useAuth } from "../../Context/Auth-context/Auth-context";
```
Comment (MiheerTamkhane) [inline src/Component/AddPlaylist/AddPlaylist.jsx:4]: you can use barrel exports for all contexts...

**Review by shubhamkumar648 — APPROVED** [review]
- pr are ready to merge
- feature is working  perfect according to given pr

**Review by MiheerTamkhane — APPROVED** [review]
Overall good work. Suggestion when clicking on the like button doesn't flip to dislike. just change its appearance to filled with color or something. Great work

File: src/App.js (hunk 1 above)
Comment (saketkum11) [reply src/App.js]: done.

File: src/Component/AddPlaylist/AddPlaylist.jsx (hunk 2 above)
Comment (saketkum11) [reply src/Component/AddPlaylist/AddPlaylist.jsx:4]: will change it.

**Review by sharathnair9999 — APPROVED** [review]

**Pull request merged**